driver/aws/s3_test.go

```diff
@@ -0,0 +1,12 @@
+package aws
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestConnecttion_ConnectAws(t *testing.T) {
+	session, err := ConnectAws()
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>too few arguments in call to ConnectAws (typecheck)